homeassistant/components/metoffice/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
```

Comment:
All the fields in the config flow must have appropriate data_descriptions that explain what the fields are